src/test/java/study/StringAddCalculator.java

```diff
@@ -0,0 +1,47 @@
+package study;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Stream;
+
+public class StringAddCalculator {
+    static int splitAndSum(String text){
+        int sum = 0;
+        if (text==null || text.isEmpty()){
+            return 0;
+        }
+        if (text.length()==1){
+            int number = Integer.parseInt(text);
+            return number;
```

Comment:
```suggestion
            return Integer.parseInt(text);
```
추가 로직이 없다면 바로 리턴해주어도 좋을 것 같네요 :)